src/ui/osx/TogglDesktop/Feature/TimeEntry/Views/TimeEntryCollectionView.m

```diff
@@ -138,8 +138,7 @@ - (void)deleteEntry
 		{
 			if (toggl_delete_time_entry(ctx, [cell.GUID UTF8String]))
 			{
-				[self deleteItemsAtIndexPaths:[[NSSet alloc] initWithArray:@[self.latestSelectedIndexPath]]];
```

Comment:
I deleted this crash code because when `toggl_delete_time_entry` is triggered, it will reload entire collection view with new Entry List.

The old code tries to remove the invalid `self.latestSelectedIndexPath` -> crash 💥